views/jira-manual-app-creation.hbs

```diff
@@ -0,0 +1,158 @@
+    <link
+      rel="stylesheet"
+      href="/public/atlassian-ui-kit/bundle.css"
+      media="all"
+    />
+    <link rel="stylesheet" href="/public/css/global.css" media="all" />
+    <link rel="stylesheet" href="/public/css/server-error-message-box.css" media="all" />
+    <link
+      rel="stylesheet"
+      href="/public/css/jira-manual-app-creation.css"
+      media="all"
+    />
+    <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
+  </head>
+
+  <body class="aui-page-hybrid">
+  {{> navigation previousPageName="Jira Select App Creation" }}
+
+  <div class="jiraManualAppCreation">
+
+      <h1>Manual app Creation</h1>
+
+      <div>Learn how to <a href="#">manually set up a GitHub app</a>. All fields are required.</div>
+
+      <form id="jiraManualAppCreation__form" class="aui top-label" enctype="multipart/form-data">
+        <div class="jiraManualAppCreation__formTitle">
+          Details
+          <i class="aui-icon aui-iconfont-question-filled"></i>
+        </div>
+
+        <div class="field-group">
+          <label for="app-name">GitHub app name</label>
+          <input class="text full-width-field" type="text"
+                 data-aui-validation-field required
+                 id="app-name" name="app-name" />
+        </div>
+
+        <div class="field-group ">
+          <label for="app-id">GitHub app ID</label>
+          <input class="text full-width-field" type="text"
+                 data-aui-validation-field required
+                 id="app-id" name="app-id" />
+        </div>
+
+        <div class="field-group">
+          <label for="client-id">GitHub client ID</label>
+          <input class="text full-width-field" type="text"
+                 data-aui-validation-field required
+                 id="client-id" name="client-id" />
+        </div>
+
+        <div class="jiraManualAppCreation__formTitle">URLs</div>
+        <div class="jiraManualAppCreation__formSubTitle">
+          Copy and paste the following URLs into your GitHub Enterprise Server configuration. This is unique to your site.
+        </div>
+
+        <div class="field-group">
+          <label for="callback-url">Callback URL</label>
+          <input class="text full-width-field" type="text" id="callback-url"
+                 name="callback-url" value="{{githubBaseUrl}}github/callback/{{uuid}}"
+                 readonly
+          />
+        </div>
+
+        <div class="field-group">
+          <label for="setup-url">Setup URL</label>
+          <input class="text full-width-field" type="text" id="setup-url"
+                 name="setup-url" value="{{githubBaseUrl}}github/setup/{{uuid}}"
+                 readonly
+          />
+        </div>
+
+        <div class="field-group">
+          <label for="webhook-url">Webhook URL</label>
+          <input class="text full-width-field" type="text" id="webhook-url"
+                 name="webhook-url" value="{{githubBaseUrl}}github/webhook/{{uuid}}"
```

Comment:
Urls won't be passed or stored with the trailing `/` so it needs to be included here after `{{githubBaseUrl}}`

Reply:
`/` has been added